schemas/input/csl-data.json

```diff
@@ -468,8 +473,19 @@
         }
       ]
     },
+    "edtf-datatype": {
+      "title": "EDTF datatype pattern",
+      "description": "CSL input supports EDTF, validated against this regular expression.",
+      "type": "string",
+      "pattern": "[0-9,\-,%,~,X,?,..,\/]"
```

Comment:
This just validates the basic characters allowed. Would be nice to have some tighter.